jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java

```diff
@@ -226,14 +352,14 @@ public void releaseTask( final Long taskId,
         taskOperationsService.call( new RemoteCallback<Void>() {
             @Override
             public void callback( Void nothing ) {
-                view.displayNotification( "Task Released" );
+                view.displayNotification( Constants.INSTANCE.TaskReleased(String.valueOf(taskId)) );
                 refreshGrid();
             }
         }, new ErrorCallback<Message>() {
             @Override
             public boolean error( Message message,
                                   Throwable throwable ) {
-                errorPopup.showMessage( "Unexpected error encountered : " + throwable.getMessage() );
+                errorPopup.showMessage( Constants.INSTANCE.UnexpectedError(throwable.getMessage()) );
```

Comment:
Errai allows us to use slf4j, see: http://docs.jboss.org/errai/latest/errai/reference/html_single/#sid-74908675
That would be my preference moving forward, instead of using GWT.log.